README.md

```diff
@@ -78,26 +78,24 @@ To compile the contracts:
 
 ---
 ### Run
-To deploy instances of the contracts for local development without prior knowledge Ganache and Truffle, run the following command:
+To deploy instances of the contracts for local development without prior knowledge Ganache and Hardhat, run the following command:
 ```shell
 ./go contracts:run
 ```
 
-This command starts up Ganache on a random port and migrates all contracts to the Ganache instance. The Ganache instance will remain running in the background, even though the command prompt returns.
+This command starts up built-in Hardhat Network and migrates all contracts to the Hardhat Network instance.
 The pid of the Ganache instance is written to a file in the run/pid directory.
 
 To stop the Ganache instance, run the following command:
 ```shell
 ./go ganache:stop
 ```
 
-If preferred by those who are familiar with Truffle and Ganache, the standard Truffle and Ganache commands can be used. Ganache can be started up manually in a terminal using the command:
-```shell
-  node ./node_modules/.bin/ganache-cli --port 8545 --allowUnlimitedContractSize --acctKeys build/ganache/accounts-8545.json
-```
+If preferred by those who are familiar with Hardhat, the standard Hardhat commands can be used. Ganache can be started up manually by configuring a local network to be run against or using the `hardhat-ganache` plugin. For more information on how this can be achieved refer to the [official Hardhat documentation](https://hardhat.org/guides/ganache-tests.html#running-tests-with-ganache)
+
 In a separate terminal, contracts can be deployed using
 ```shell
-  ./node_modules/.bin/truffle migrate
+  npx hardhat --network [customNetworkName] deploy
 ```
 One of the contracts that gets deployed locally is a mock contract that represents the $BOSON token. The mock exists for unit testing purposes and so that those who want to develop against the protocol locally don't have to point to a testnet deployment of the $BOSON token.
 
```

Comment:
I think here we should mention that if you want to deploy to a testnet like Rinkeby, .env.example needs to be copied to .env and the values provided

Reply:
I am not sure if this is the most appropriate place as this entire section is related to the Local development. From what i see we don't have any procedure steps for a testnet deployment in the README. I could add a few sentences, but this should be on a separate PR IMO

Reply:
OK. Let's revisit after we merge. There are some changes to the README regarding running on Windows that were never merged and probably need to be, so we could do another pass at the README at a later date.